purchase_request/models/purchase_request_line.py

```diff
@@ -44,7 +44,6 @@ class PurchaseRequestLine(models.Model):
         comodel_name="res.company",
         related="request_id.company_id",
         string="Company",
-        store=True,
```

Comment:
Why removing the store? In fact, I would do it stored with index for ir.rule faster application.

Reply:
IMO there is no sense in a related store field (in that case it should be a compute store), the `company_id` value of the lines will always be that of the header (a line will not be defined for a different company) therefore, the appropriate thing to do is to define the value as related no store.

Reply:
The sense is for doing searches over that field in a performant way. For example, for the record rules application (the multi-company rule), so please restore it and put the index.